### PR TITLE
percona-toolkit: build with latest mysql-client

### PR DESCRIPTION
**Pull request by bevanjkay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/p/percona-toolkit.rb [hunk 1]
```diff
@@ -45,8 +43,8 @@ class PerconaToolkit < Formula
   end
 
   resource "DBD::mysql" do
-    url "https://cpan.metacpan.org/authors/id/D/DV/DVEEDEN/DBD-mysql-5.002.tar.gz"
-    sha256 "8dbf87c2b5b8eaf79cd16507cc07597caaf4af49bc521ec51c0ea275e8332e25"
+    url "https://cpan.metacpan.org/authors/id/D/DV/DVEEDEN/DBD-mysql-5.004.tar.gz"
```
Comment (chenrui333) [inline Formula/p/percona-toolkit.rb:46]: not quite sure why it is 5.004 rather than 5.005.

Comment (bevanjkay) [reply Formula/p/percona-toolkit.rb:46]: 5.004 is the latest version? https://github.com/perl5-dbi/DBD-mysql/releases

Comment (chenrui333) [reply Formula/p/percona-toolkit.rb:46]: somehow this commit did not get tagged to 5.005, https://github.com/perl5-dbi/DBD-mysql/commit/21e9ef2cd0d8439944003d32bcbd592ccb8326d7

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
just some upstream tag question before the merge

File: Formula/p/percona-toolkit.rb (hunk 1 above)
Comment (bevanjkay) [reply Formula/p/percona-toolkit.rb:46]: Not sure if it affects other functionality, but the functions causing previous build failures where patched in 5.004 https://github.com/perl5-dbi/DBD-mysql/pull/415

**Pull request merged**